xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -734,12 +734,15 @@ CGUIFontTTF::Character* CGUIFontTTF::GetCharacter(character_t chr, FT_UInt glyph
   wchar_t letter = (wchar_t)(chr & 0xffff);
   character_t style = (chr & 0x7000000) >> 24;
 
+  if (!glyphIndex)
+    glyphIndex = FT_Get_Char_Index(m_face, letter);
+
   // ignore linebreaks
   if (letter == L'\r')
     return nullptr;
 
   // quick access to ascii chars
-  if (letter < 255 && glyphIndex < 255)
```

Comment:
About the remove of `letter < 255` check, can you confirm with other write languages (Israel, arabic, china) that no bad problems come.

Reply:
No problem, I can leave the `letter < 255` checks in.

Reply:
It's in again.